.circleci/config.yml

```diff
@@ -212,7 +212,13 @@ jobs:
     executor: node
     steps:
       - checkout
-      - run: ./scripts/assert-changed-files.sh "packages/*|.circleci/*"
+      # We should always assert on master
+      - when:
+          condition:
+            - not:
+                equal: [master, << pipeline.git.branch >>]
```

Comment:
can you explain what this is doing?

Reply:
always run this on master, else use the packages detection skip